### DISCUSSION TITLE
Filecheck Configuration

### DISCUSSION THREAD

**Discussion by jorwern** [Geektoolkit/Dynaframe3#66] [Q&A]
Dear all,

where can i set how often Dynaframe3 will look for new files in the directory. Currently i am using Pi3D but would like to change to Dynaframe, however I would like to check where this configuration is. Further, what would happen if new files where found, will automatically a new playlist be generated? Many thanks in advance and best regards

John

**Comment by Geektoolkit** [comment]
Currently the behavior is it gets the file list, and when it is done going through the list it refreshes the list. Changes to settings that would affect the file list (such as turning on sorting) will also refresh the list.  There's also a command to refresh the list...if you select that you'll see in the URL the command that you can use to do this whenver you want...if you have some scripting knowlege you can execute a get request to that http url and force the refresh as necessary.

I don't have scheduled refreshes currently. I'll have to look to see if I can detect a file system change to pick up new files.
